### PR TITLE
Add pretty focus states!

### PR DESCRIPTION
**Pull request by notwaldorf**
Adding behaviours to the radio button!

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata @morethanreal  PTAL

**Comment by notwaldorf** [comment]
I signed it!

File: paper-radio-button.html [hunk 1]
```diff
@@ -147,29 +121,13 @@
...
-        return;
-      }
-
-      var old = this.checked;
-      this.checked = !this.isToggle || !this.checked;
-
-      if (this.checked !== old) {
-        this.fire('change');
-      }
+    _buttonStateChanged: function() {
```
Comment (cdata) [inline paper-radio-button.html:124]: I still think that overriding private implementation of a behavior is a potentially brittle practice :fearful:

**Comment by cdata** [comment]
One bikeshed convention, otherwise LGTM :+1:

File: paper-radio-button.html (hunk 1 above)
Comment (notwaldorf) [inline paper-radio-button.html:124]: @sjmiles what do you think? I just based this on your initial implementation of `paper-toggle-button`

(going to merge this and let future monica deal with the answer in a different PR though)

**Pull request merged**